README.md

```diff
@@ -11,18 +11,37 @@ to their version constraints) packages into a Composer Repository file.
 Usage
 -----
 
-- Download [Composer](https://getcomposer.org/download/): `curl -sS https://getcomposer.org/composer.phar -o composer.phar`
-- Install satis: `php composer.phar create-project composer/satis --stability=dev --keep-vcs`
+- Install satis: `php composer.phar create-project composer/satis:dev-master --keep-vcs`
```

Comment:
most projects are showing the usage of a global ``composer`` rather than ``php composer.phar``, especially when you remove the download step